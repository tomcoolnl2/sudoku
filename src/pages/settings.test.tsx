
import { render } from '@testing-library/react'
import { withTheme, withRedux, withRouter } from '../test'
import { Settings } from '.'

describe('Settings page', () => {
	
	it('should match it\'s snapshot', () => {
		const { container } = render(withTheme(withRedux(withRouter(<Settings />))))
		expect(container.firstChild).toMatchSnapshot()
	})
})