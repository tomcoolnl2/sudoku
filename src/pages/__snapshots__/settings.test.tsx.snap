// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings page should match it's snapshot 1`] = `
<a
  href="/game"
>
  <button
    type="button"
  >
    Back to the game
  </button>
</a>
`;
